src/components/table.js

```diff
@@ -259,37 +280,52 @@ FlexTable.propTypes = {
   headerHeight: PropTypes.number,
   rowHeight: PropTypes.number,
   styleHeader: PropTypes.func,
-  styleCell: PropTypes.func
+  styleCell: PropTypes.func,
+  tableName: PropTypes.string,
+  sort: PropTypes.shape({
+    field: PropTypes.string,
+    direction: PropTypes.string
+  })
 }
 
 /**
  * A basic table with a header and flexible column widths. Intended for small amounts of data,
  * since it does not provide scrolling. See FlexTable for prop types.
  */
-export const SimpleFlexTable = ({ columns, rowCount, noContentMessage, hoverHighlight }) => {
-  return h(Fragment, [
-    div({ style: { height: 48, display: 'flex' } }, [
-      _.map(([i, { size, headerRenderer }]) => {
+export const SimpleFlexTable = ({ columns, rowCount, noContentMessage, hoverHighlight, tableName = 'data table', sort = null }) => {
+  return div({
```

Comment:
It seemed like `Fragment` doesn't actually render any markup on its own, it's just a container if you have more than one child. I changed it to a `div` since I needed to add ARIA properties and needed an actual tag to attach them to.